core/src/main/java/org/bitcoinj/core/TransactionConfidence.java

```diff
@@ -82,7 +83,8 @@ public TransactionConfidence createConfidence(Sha256Hash hash) {
      */
     private CopyOnWriteArrayList<PeerAddress> broadcastBy;
     /** The time the transaction was last announced to us. */
-    private Date lastBroadcastedAt;
+    @Nullable
+    private Instant lastBroadcastedAt = null;
```

Comment:
Since this is private and the type is changing maybe rename it to `lastBroadcastTime` or `lastBroadcastInstant`?

Reply:
Will do.